analyzers/src/SonarAnalyzer.CFG/packages.lock.json

```diff
@@ -459,13 +449,13 @@
       },
       "Microsoft.NETCore.Platforms": {
         "type": "Transitive",
-        "resolved": "1.1.1",
-        "contentHash": "TMBuzAHpTenGbGgk0SMTwyEkyijY/Eae4ZGsFNYJvAr/LDn1ku3Etp3FPxChmDp5HHF3kzJuoaa08N0xjqAJfQ=="
+        "resolved": "1.1.0",
```

Comment:
Downgrade. Most likely caused by the old version of the analyzer Microsoft.CodeAnalysis.CSharp.Workspaces referenced.